crates/store/re_protos/proto/rerun/v1alpha1/cloud.proto

```diff
@@ -271,6 +286,29 @@ message ScanPartitionTableResponse {
   rerun.common.v1alpha1.DataframePart data = 1;
 }
 
+message GetDatasetManifestSchemaRequest {
+  reserved "dataset_id";
```

Comment:
Not sure what's the reasoning with this and other reservations below? These types never existed, did they? I.e. there's nothing to be BW compatible with?
```suggestion
```